components/spluseins-calendar.vue

```diff
@@ -12,131 +12,65 @@
 import * as moment from 'moment';
 import { Calendar, Day, Units } from 'dayspan';
 import { mapMutations, mapState, mapGetters, mapActions } from 'vuex';
-import colors from 'vuetify/es5/util/colors';
 import DayspanCustomCalendar from './dayspan-custom-calendar.vue'
 
-const hashCode = (string) =>
-string.split('').reduce((prevHash, currVal) =>
-  (((prevHash << 5) - prevHash) + currVal.charCodeAt(0))|0, 0);
-
-
 export default {
   name: 'SpluseinsCalendar',
   layout: 'empty',
   components: {
     DayspanCustomCalendar
   },
   data() {
-    const around = Day.fromMoment(moment().startOf('isoWeek'));
-    const calendarWeekType = {
+    const startOfWeek = Day.fromMoment(moment().startOf('isoWeek'));
+    const weeklyCalendar = {
       id: 'W',
       label: 'Woche',
       shortcut: 'W',
       type: Units.DAY,
       size: 7,
-      around,
+      around: startOfWeek,
       focus: 0,
       repeat: true,
       listTimes: true,
       updateRows: true,
       schedule: false
     };
-    const calendar = Calendar.days(7, around, 0);
+   const calendar = Calendar.days(7, startOfWeek, 0);
+
+    // computed properties are not available during client rendering yet, access the getter directly
+    calendar.setEvents(this.$store.getters['splus/getLecturesAsEvents']);
```

Comment:
gute Auslagerung der Logik

Reply:
`this.events` ging nicht im Client, obwohl es über `computed` auf `splus/getLecturesAsEvents` durchreichen sollte. Auf dem Server geht es. Vielleicht ist das ein Bug. Das will ich noch mal recherchieren.